examples/tests/valid/cors_api.w

```diff
@@ -0,0 +1,132 @@
+bring cloud;
+bring ex;
+bring http;
+bring "./assertions.w" as t;
+
+
+// DEFAULT CORS
+// ============
+// without specifying cors options, the default is to allow all origins
+let apiDefaultCors = new cloud.Api(
+  cors: {}
```

Comment:
don't like this, would prefer to have `cors: true`, or `cors: {...}` but doesn't look like it's possible in the type system. What do you think about it? Would be two distinct properties better?

Reply:
Yes, two properties.

`cors: bool` and `corsOptions`

Reply:
done

Reply:
You probably want to update this test